.github/workflows/kind.yml

```diff
@@ -68,6 +89,52 @@ jobs:
         path: flow-aggregator.tar
         retention-days: 1 # minimum value, in case artifact deletion by 'artifact-cleanup' job fails
 
+  test-e2e-ubi-basic:
+    name: Basic e2e tests on a Kind cluster for the UBI image
+    needs: [build-antrea-ubi-image]
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Free disk space
+      # https://github.com/actions/virtual-environments/issues/709
+      run: |
+        sudo apt-get clean
+        df -h
+    - uses: actions/checkout@v4
+      with:
+        show-progress: false
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: 'go.mod'
+    - name: Download Antrea image from previous job
+      uses: actions/download-artifact@v3
+      with:
+        name: antrea-ubi
```

Comment:
Given that we only want 1 test job for the UBI image, we probably don't need a separate job to build the image, and we don't need to upload / download the image as an artifact. It usually adds some overhead. What do you think?

Reply:
Indeed. I have combined the test with the build.yml.